ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/valueprovider/CascadingDefaultProvider.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
do we want to move this to Signer repo? (assuming its a cut-and paste from Besu)